source/index.ts

```diff
@@ -176,6 +178,7 @@ namespace is { // tslint:disable-line:no-namespace
 	export const dataView = isObjectOfType<DataView>(TypeName.DataView);
 
 	export const directInstanceOf = <T>(instance: any, klass: Class<T>): instance is T => Object.getPrototypeOf(instance) === klass.prototype;
+	export const urlInstance = (value: any): value is URL => !nullOrUndefined(value) && directInstanceOf<URL>(value, URL);
```

Comment:
Using instance checking will not work across realms (frames in the browser or `vm` module in Node.js). 

A better solution would be to add `URL` as a type name here: https://github.com/arturovt/is/blob/7e71742a0f6aac5f44be0f2e00ed1619c75fd557/source/index.ts#L53 and then use `isObjectOfType`, like, for example, the `is.generatorFunction` check.

Reply:
@sindresorhus Agree